src/js/components/BrimToaster.tsx

```diff
@@ -7,6 +7,7 @@ const BrimToaster = () => {
     <Toaster
       position="bottom-right"
       toastOptions={{
+        role: "status",
```

Comment:
This one is legit: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/status_role